libs/ha/ha_common_libs_k8s.py

```diff
@@ -1795,7 +1806,11 @@ def iam_bucket_cruds(self, event, s3_obj=None, user_crud=False, num_users=None,
                 if len(del_users) > i_i:
                     LOGGER.debug("Deleting %s user", del_users[i_i])
                     user = del_users[i_i]
-                    resp = self.delete_s3_acc_buckets_objects({user: del_users_dict[user]})
+                    if not header:
+                        resp = self.delete_s3_acc_buckets_objects({user: del_users_dict[user]})
+                    else:
+                        resp = self.delete_iam_user_with_header({user: del_users_dict[user]},
```

Comment:
same as above

Reply:
same as above..

Reply:
> same as above

Same here too.
We have to update test to get the header as using create_account_users() we are not passing any header value taken before.